doc/reference/nhibernate_caches.md

```diff
@@ -0,0 +1,715 @@
+# NHibernate.Caches
+
+> **NHibernate.Caches namespace contains several second-level cache
+> providers for NHibernate..**
```

Comment:
Double dot. Should we check paragraph title? It seems like some logic has added a dot to them.

And the abstract title, `What is NHibernate.Caches?`, is missing.

Reply:
Fixed. I don't know where that double dot came from. I haven't encountered it elsewhere